looprpc/client.proto

```diff
@@ -339,7 +339,7 @@ message TermsResponse {
     /**
     On-chain cltv expiry delta
     */
-    int32 cltv_delta = 7;
+    int32 cltv_delta = 7 [deprecated=true];
```

Comment:
We don't set those values anymore, so we could just as well remove them in the proto completely and only leave a `reserved`?

Reply:
That would break users of the protos, but maybe that's better than having them keep using values not set anymore?

Reply:
Did change to reserved